src/web/basics/button/tests/tests-button.html

```diff
@@ -10,11 +10,12 @@
         </style>
     </head>
     <body>
-
-        <div class='regular-button buttonHolder'></div>
-        <div class='dispose-button buttonHolder'></div>
-        <div class='updated-button buttonHolder'></div>
-        <div class='destroy-button buttonHolder'></div>
+        <div class='test-container' style="padding: 40px; margin: 0;">
```

Comment:
Does it look weird if you just capture `body`?

I think we captured '.test-container' rather than `body` on old Orion because there was a bunch of other stuff on the screen we wanted to exclude from the screenshot.

If this extra markup doesn't buy us anything than consider losing it.